core/lib/external_price_api/src/forced_price_client.rs

```diff
@@ -1,16 +1,23 @@
-use std::num::NonZeroU64;
+use std::{
+    cmp::{max, min},
+    num::NonZeroU64,
+};
 
 use async_trait::async_trait;
 use rand::Rng;
+use tokio::sync::RwLock;
 use zksync_config::configs::ExternalPriceApiClientConfig;
 use zksync_types::{base_token_ratio::BaseTokenAPIRatio, Address};
 
 use crate::PriceAPIClient;
 
-// Struct for a a forced price "client" (conversion ratio is always a configured "forced" ratio).
-#[derive(Debug, Clone)]
+const NEXT_VALUE_VARIATION_RANGE: f64 = 0.03; //3%
```

Comment:
Lets keep it as a configurable parameter please. Different testnets might want their variations to be different.

Reply:
done